autometa/common/external/samtools.py

```diff
@@ -22,17 +22,16 @@
 
 import logging
 import os
-import subprocess
 import shutil
+import subprocess
 import tempfile
 
-import pandas as pd
 import multiprocessing as mp
 
 logger = logging.getLogger(__name__)
 
 
-def sort(sam, bam, nproc=mp.cpu_count()):
+def sort(sam, bam, cpus=mp.cpu_count()):
```

Comment:
Have you tested this?

Reply:
I have tested this when running as a standalone module. I also changed `nproc` to `cpus` in the `coverage.py` script where sort was being used.